shipyard-controller/debug-ui/fetch_blockingsequences.js

```diff
@@ -0,0 +1,17 @@
+function fetchBlockingSequences(project, context, stage, blockingSequencesList) {
+  fetch(`/sequence/project/${project}/shkeptncontext/${context}/stage/${stage}/blocking`, {
+    method: "get",
+  })
+    .then((res) => {
+      return res.json();
+    })
+    .then((response) => {
+      response.forEach((blockingSequence) => {
+        if (blockingSequence.scope !== null) {
+          let li = document.createElement("li");
+            li.innerHTML = blockingSequence.scope.keptnContext
+            blockingSequencesList.append(li)
+        }
+      });
+    });
```

Comment:
there should be a `.catch( err => {} )` in case we cannot process the response due to response != 200